webapp/pages/post/_id/_slug/index.spec.js

```diff
@@ -31,7 +31,7 @@ describe('PostSlug', () => {
       $filters: {
         truncate: a => a,
       },
-      // If you mocking router, than don't use VueRouter with lacalVue: https://vue-test-utils.vuejs.org/guides/using-with-vue-router.html
+      // If you are mocking the router, then don't use VueRouter with localVue: https://vue-test-utils.vuejs.org/guides/using-with-vue-router.html
```

Comment:
That’s an unrelated file !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
😂

Reply:
!!!111eleven :angry:

Reply:
what is this supposed to mean? hahaha

Reply:
When people get really angry, they use a lot of exclamation marks and don't manage to keep the shift key pressed!!!11